executor/parallel_NLJ.go

```diff
@@ -0,0 +1,318 @@
+// Copyright 2020 PingCAP, Inc.
```

Comment:
how about renaming the file to `parallel_nested_loop_apply`?

Reply:
It is renamed to `parallel_apply` now.